hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -101,7 +102,7 @@ public class SCMSecurityProtocolServer implements SCMSecurityProtocol,
       .getLogger(SCMSecurityProtocolServer.class);
   private final CertificateServer rootCertificateServer;
   private final CertificateServer scmCertificateServer;
-  private final X509Certificate rootCACertificate;
+  private final List<X509Certificate> rootCACertificate;
```

Comment:
Can we change the name to add suffix/prefix "list" to avoid confusion that it not just single Certificate.